src/onegov/core/utils.py

```diff
@@ -1226,3 +1228,35 @@ def batched(
             return
 
         yield batch
+
+
+def generate_fts_phonenumbers(numbers: Iterable) -> list[str]:
+    """
+    Generates a list of phonenumbers in various formats for full text search.
+    The international, the national and the local format as well as the
+    extension.
+
+    """
+    result = []
+
+    for number in numbers:
+        if number:
+            try:
+                parsed = parse(number, 'CH')
+            except NumberParseException:
+                # allow invalid phone number
+                result.append(number.replace(' ', ''))
+                continue
+
+            result.append(format_number(
+                parsed, PhoneNumberFormat.E164))
+
+            national = format_number(
+                parsed, PhoneNumberFormat.NATIONAL)
+            groups = national.split()
+            for idx in range(len(groups)):
+                partial = ''.join(groups[idx:])
+                if len(partial) > 3:
+                    result.append(partial)
```

Comment:
```suggestion
        if not number:
            continue

        try:
            parsed = parse(number, 'CH')
        except NumberParseException:
            # allow invalid phone number
            result.append(number.replace(' ', ''))
            continue

        result.append(format_number(
            parsed, PhoneNumberFormat.E164))

        national = format_number(
            parsed, PhoneNumberFormat.NATIONAL)
        groups = national.split()
        for idx in range(len(groups)):
            partial = ''.join(groups[idx:])
            if len(partial) > 3:
                result.append(partial)
```
Let's dedent this by one level